docker/build_scripts_pypy/install_pypy.sh

```diff
@@ -18,6 +18,11 @@ function install_one_pypy {
     cd /opt/pypy
     tar xf $tarball
 
+    # the new PyPy 3 distributions don't have pypy symlinks to pypy3
+    if [ ! -f "$outdir/bin/pypy" ]; then
+        ln -s pypy3 $outdir/bin/pypy
+    fi
+
```

Comment:
Hmm, I guess this is something we should do in the packaging script.

Reply:
Unless it was intentionally so so one can unpack both 2.7 and 3.x in the same directory without clashes...